src/models/model.js

```diff
@@ -0,0 +1,28 @@
+import { pool } from './pool';
+
+class Model {
+  constructor(table) {
+    this.pool = pool;
+    this.table = table;
+    this.pool.on('error', (err, client) => `Error, ${err}, on idle client${client}`);
+  }
+
+  async select(columns, clause) {
+    let query = `SELECT ${columns} FROM ${this.table}`;
+
+    if (clause) query += clause;
+    console.log(query, '---> ');
```

Comment:
Please let's remove this unnecessary console statement, it's a good idea to check your code and remove any unused or debug codes when pushing for review.